src/client/Client.cc

```diff
@@ -6019,7 +6023,8 @@ int Client::may_delete(Inode *dir, const char *name, const UserPerm& perms)
   return r;
 }
 
-int Client::may_delete(const char *relpath, const UserPerm& perms) {
+int Client::may_delete(const char *relpath, const UserPerm& perms)
+{
```

Comment:
this change looks unrelated, i hope this is fine?

Reply:
It's okay. I just fixed the code style to make it the same with all the other we did.